docs/guides/unity-catalog-aws.md

```diff
@@ -4,7 +4,7 @@ page_title: "Unity Catalog set up on AWS"
 
 # Deploying pre-requisite resources and enabling Unity Catalog (AWS Preview)
 
--> **Private Preview** This feature is in [Private Preview](https://docs.databricks.com/release-notes/release-types.html). Contact your Databricks representative to request access. 
+-> **Public Preview** This feature is in [Public Preview](https://docs.databricks.com/data-governance/unity-catalog/index.html). Contact your Databricks representative to request access. 
```

Comment:
please keep it as "private preview" for now, unless Paul says otherwise.

Reply:
our public preview docs are now out, so let's keep it consistent 😄